src/kmergraph.cpp

```diff
@@ -261,8 +492,9 @@ void KmerGraph::set_nb(const float &nb_prob, const float &nb_fail) {
     nb_r += nb_fail;
 }
 
-float KmerGraph::nb_prob(uint32_t j, const uint32_t &sample_id) {
-    auto k = nodes[j]->get_covg(0, sample_id) + nodes[j]->get_covg(1, sample_id);
+
+float KmerGraphWithCoverage::nb_prob(uint32_t j, const uint32_t &sample_id) {
```

Comment:
Rename `j` to something more descriptive

Reply:
I can do this in my commit - changing these functions

Reply:
updated in commit bec1ff33d23fb4178752ee6a586f07ccd6d9a0d0